pkg/gardenlet/controller/backupentry/backup_entry_reconciler.go

```diff
@@ -73,98 +73,104 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 	be := &gardencorev1beta1.BackupEntry{}
 	if err := gardenClient.Client().Get(ctx, request.NamespacedName, be); err != nil {
 		if apierrors.IsNotFound(err) {
-			r.logger.Debugf("[BACKUPENTRY RECONCILE] %s - skipping because BackupEntry has been deleted", request.NamespacedName)
+			log.Info("Object is gone, stop reconciling")
```

Comment:
Nit: to keep my not found comments consistent 
```suggestion
			log.V(1).Info("Object is gone, stop reconciling")
```

Reply:
Same like https://github.com/gardener/gardener/pull/6266#discussion_r914710398